steps/expval.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The PennyLane-Orquestra step for computing the expectation value of Hermitian
+operators given a quantum circuit and a quantum device.
```

Comment:
Not quite like this. The step that they have for the VQE tutorial incorporates other Orquestra interfaces (e.g., optimizers, etc.) and accepts ansatze that are pre-defined rather than dynamic (e.g., `SingletUCCSDAnsatz`):
* [VQE tutorial](https://github.com/zapatacomputing/z-quantum-vqe/blob/master/examples/hydrogen-vqe.yaml)
* [VQE step used to optimize](https://github.com/zapatacomputing/z-quantum-optimizers/blob/88de904dd5ee2ce58d51d0a6a59ad8afe31aa28b/steps/optimizers.py#L14)

Internally, these functions are used, however, to compute expectation values.

Reply:
Got it!